tools/packaging/packaging.psm1

```diff
@@ -106,6 +110,14 @@ function Start-PSPackage {
         }
     }
 
+    if($Configuration -ine 'Release' -and $NameSuffix)
+    {
+        $NameSuffix = "$Configuration-$NameSuffix"
+    }
+    elseif ($Configuration -ine 'Release') {
+        $NameSuffix = $Configuration
```

Comment:
Yes, but, I'll update this code, as when I wrote it I assumed `Release` was the only Release configuration